applications/rag/example_notebooks/rag-kaggle-ray-sql-latest.ipynb

```diff
@@ -284,16 +284,14 @@
     ")\n",
     "\n",
     "print(\"Job submitted with ID:\", job_id)\n",
-    "prev_status = \"\"\n",
     "while True:\n",
     "    status = client.get_job_status(job_id)\n",
-    "    if status != prev_status:\n",
-    "        print(\"Job status:\", status)\n",
-    "        print(\"Job info:\", client.get_job_info(job_id).message)\n",
-    "        prev_status = status\n",
+    "    print(\"Job status:\", status)\n",
+    "    print(\"Job info:\", client.get_job_info(job_id).message)\n",
     "    if status.is_terminal():\n",
     "        break\n",
-    "    time.sleep(1)\n",
+    "    time.sleep(10)\n",
```

Comment:
10s too frequent? 30s?

Reply:
yeah, probably 30s since the job takes 5-10 mins to finish. maybe we can also add a comment saying "job may take 5-10 mins to run"?